### ISSUE TITLE
Add Android Material You icon

### ISSUE THREAD

**Issue by Darkempire78** [erickok/transdroid#628]
https://developer.android.com/develop/ui/views/launch/icon_design_adaptive

Since android 13, icons should follow Material You

**Comment by erickok** [comment]
Do you mean the themed icons? We'd have to switch to a monochrome app icon though. Maybe that's good to do, yes.

**Comment by Darkempire78** [comment]
Yes a themed icon

**Comment by bwitt** [comment]
@erickok do you have the launcher logo at higher resolutions, without the shadow?  I can take a crack at this but looks like I'll need higher resolutions and without the shadow.  I can scale it up but might not look as good,  and will still have the shadow.

I think each density needs these sizes for the logo itself:
hdpi 108
xhdpi 144
xxhdpi 216
xxxhdpi 288

**Comment by erickok** [comment]
Yes I have the original XCF (!) file and also a PSD with 'banner' that has the gear icon included.
[assets.zip](https://github.com/erickok/transdroid/files/10061020/assets.zip)

**Comment by bwitt** [comment]
Thanks for sending that!  I forgot to ask, when I open the XCF file in gimp the colors seem different than what I see in the app.  Is there something I need to configure in gimp to fix?

**Comment by Darkempire78** [comment]
any update?

**Comment by bwitt** [comment]
I haven't had a chance to work on this; if someone else wants to work on this that'd be fine by me!
